.circleci/config.yml

```
@@ -23,7 +23,7 @@ anchor_1: &attach_options
 anchor_2: &filter_release_tag
   filters:
     tags:
-      only: /^v\d+\.\d+\.\d+$/
+      only: /^v\d+\.\d+\.\d+(?:-rc\.\d)?$/
```

```suggestion
      only: /^v\d+\.\d+\.\d+(?:-rc\d+)?$/
```

---

```suggestion
      only: /^v\d+\.\d+\.\d+(?:-rc\.?\d+)?$/
```

---

Why that? 🤔

---

the PC uses the dot between rc and digit; although our latest pre-release doesn't, it was the first one, so I thought just allowing that on this regex wouldn't hurt. To be more precise, actually, I'd prefer a regex which would match ANY valid [pre-release tag from semver](https://semver.org/#spec-item-9)

---

From [semver's docs](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string)
```suggestion
      only: /^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/
```

---

Amend: our pre-release has the dot, so we should comply

---

Alright. So it should be your last expression from `semvers` plus the optional dot?

---

@weilbith The above regex change that you see is directly coming from the [semver docs](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string) and satisfies all the semver release, pre-release criteria. Hence I will go with it.